wandb/sdk/internal/internal_api.py

```diff
@@ -1702,7 +1702,16 @@ def no_retry_4xx(e):
             ):
                 return True
             body = json.loads(e.response.content)
-            raise UsageError(body["errors"][0]["message"])
+            message = body["errors"][0]["message"]
+            if (
+                e.response.status_code == 400
+                and "Sweep" in message
```

Comment:
i'm not loving the specific workaround of using the string content to figure out this case. 
Is there a better way to get this indication? Or maybe just change the error message?

Reply:
I actually agree with @kptkin, @vwrj. Is there a better way to do this?